### PR TITLE
Added getting device's filesystem

### PR DESCRIPTION
**Pull request by Deixx**
Signed-off-by: Daniel Madej <daniel.madej@intel.com>

### CODE REVIEW

File: storage_devices/device.py [hunk 1]
```diff
@@ -13,7 +14,7 @@ class Device:
     def __init__(self, path):
         self.size = Size(disk_utils.get_size(path.replace('/dev/', '')), Unit.Byte)
         self.system_path = path
-        self.filesystem = None
+        self.filesystem = get_device_filesystem_type(path)
```
Comment (Ostrokrzew) [inline storage_devices/device.py]: Debugger says that `path` is "/dev/XXX" but in this function You need only the end of path - this XXX. Change argument to `path.replace('/dev/', '')`
or to `path.split('/')[2]`
or modify `cmd = f'lsblk -l -o NAME,FSTYPE | grep "{device_system_path} "'` in called method similar way.

File: test_tools/disk_utils.py [hunk 2]
```diff
@@ -309,3 +309,17 @@ def wipe_filesystem(device, force=True):
     TestRun.executor.run_expect_success(cmd)
     TestRun.LOGGER.info(
         f"Successfully wiped filesystem from device: {device.system_path}")
+
+
+def get_device_filesystem_type(device_system_path):
+    cmd = f'lsblk -l -o NAME,FSTYPE | grep "{device_system_path} "'
```
Comment (Ostrokrzew) [inline test_tools/disk_utils.py]: whitespace

File: storage_devices/device.py (hunk 1 above)
Comment (Deixx) [reply storage_devices/device.py]: Done

File: test_tools/disk_utils.py (hunk 2 above)
Comment (Deixx) [reply test_tools/disk_utils.py]: this whitespace is intentional to eliminate partitions from output if a device with partitions is checked

**Pull request merged**